patchwerk-sim.py

```diff
@@ -164,11 +221,18 @@ def run_simulation():
     number_simulations = int(args.sims)
     number_survived = 0
     overhealing_list = []
+    
+    tanks = [
+        Tank(name='Bearly', max_health=13000, dodge_parry=0.2, mitigation=0.75),
+        Tank(name='Zug Zug', max_health=11000, dodge_parry=0.3, mitigation=0.7),
```

Comment:
it's likely warrior miti is 73% with good mitigation gear (including bwl or aq40 shield), in defensive stance, assuming inspiration is up, along with all other standard consumable buffs.

a geared offtank will also have about 36% mitigation chance (I think), but maybe i'm overestimating. from sixtyupgrades.com (https://sixtyupgrades.com/set/9rcYYsm67yxa4BBFhenE3J) and from https://github.com/magey/classic-warrior/wiki/Attack-table#miss, i get this calculation:

```
> misschance = 5 + (376-315) * 0.04
> dodgechance = 19.54
> parrychance = 9.04
> avoidchance = misschance + dodgechance + parrychance
> avoidchance
[1] 36.02
```

if you think it's overestimate based on logs, maybe we can cut it down to 33%. 9.5k hp no-wbuffs (with zanza) also seems best based on lubs' figures.

for druid bear i'm pretty sure we've failed to count the miss chance from +defense skill. we should use 25% or 26% avoid chance on a bear. did doodoo say what max hp was with no wbuffs?